diff --git a/kernel-6.6/KernelSU-Next/kernel/allowlist.c b/kernel-6.6/KernelSU-Next/kernel/allowlist.c
index e7375e5d..0a172220 100644
--- a/kernel-6.6/KernelSU-Next/kernel/allowlist.c
+++ b/kernel-6.6/KernelSU-Next/kernel/allowlist.c
@@ -327,14 +327,6 @@ bool ksu_uid_should_umount(uid_t uid)
 #ifndef CONFIG_KSU_DISABLE_POLICY
     struct app_profile profile = { .current_uid = uid };
 #endif
-    if (unlikely(is_uid_manager(uid))) {
-        // we should not umount on manager!
-        return false;
-    }
-    if (unlikely(uid == WEBVIEW_ZYGOTE_UID)) {
-        // we should not umount for webview zygote
-        return false;
-    }
 #ifdef CONFIG_KSU_DISABLE_POLICY
     return !__ksu_is_allow_uid(uid);
 #else
